data/processed/hesa/total_postgraduates.yaml

```diff
@@ -0,0 +1,22 @@
+api: fetch
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/students
+description: Number of postgraduate (research) students enrolled full-time in universities in a NUTS2 region in the starting academic year.
+endpoint_static: https://www.hesa.ac.uk/data-and-analysis/students/table-13.csv
+framework_group: public_rnd_capability
+is_experimental: False
+nuts_level: 2
```

Comment:
We need to add the line:
```
order: [year, nuts_id, nuts_year_spec, value.id]
```
see #131 and #130.

Reply:
Done

Reply:
Done